packager/live_packager_test.cc

```diff
@@ -1215,20 +1290,29 @@ INSTANTIATE_TEST_CASE_P(
         LivePackagerReEncryptCase{
             7, "encrypted/prd_data/init.mp4",
             LiveConfig::EncryptionScheme::CENC, LiveConfig::OutputFormat::FMP4,
-            LiveConfig::TrackType::VIDEO, "encrypted/prd_data/%05d.m4s", true},
+            LiveConfig::TrackType::VIDEO,
+            ProtectionSystem::kPlayReady | ProtectionSystem::kWidevine,
+            "encrypted/prd_data/%05d.m4s", true, true},
         // Verify decrypt FMP4 and re-encrypt to FMP4 with CBCS encryption,
         // ENABLE processing EMSG.
         LivePackagerReEncryptCase{
             7, "encrypted/prd_data/init.mp4",
             LiveConfig::EncryptionScheme::CBCS, LiveConfig::OutputFormat::FMP4,
-            LiveConfig::TrackType::VIDEO, "encrypted/prd_data/%05d.m4s", true},
+            LiveConfig::TrackType::VIDEO, ProtectionSystem::kWidevine,
+            "encrypted/prd_data/%05d.m4s", true, true},
         // Verify decrypt FMP4 and re-encrypt to FMP4 with CBCS encryption,
         // DISABLE processing EMSG
-        LivePackagerReEncryptCase{7, "encrypted/prd_data/init.mp4",
-                                  LiveConfig::EncryptionScheme::CBCS,
-                                  LiveConfig::OutputFormat::FMP4,
-                                  LiveConfig::TrackType::VIDEO,
-                                  "encrypted/prd_data/%05d.m4s", false}));
+        LivePackagerReEncryptCase{
+            7,
+            "encrypted/prd_data/init.mp4",
+            LiveConfig::EncryptionScheme::CBCS,
+            LiveConfig::OutputFormat::FMP4,
+            LiveConfig::TrackType::VIDEO,
+            ProtectionSystem::kPlayReady,
```

Comment:
Please include a test case with multiple DRM systems

Reply:
I am already coving that as part of a test: https://github.com/Pluto-tv/shaka-packager-live/blob/feature/TRANS-5763-include-pssh/packager/live_packager_test.cc#L1292